package.json

```diff
@@ -58,25 +65,24 @@
   },
   "watches": {
     "doc": [
```

Comment:
based on my read [here](https://docs.npmjs.com/files/package.json) i still have no idea what the `watches` field is or how it would be able to watch/rebuild when changes in the source are detected.